### PR TITLE
misc: rebrand project to TR1X

### PR DESCRIPTION
**Pull request by rr-**
#### Checklist

- [X] I have read the [coding conventions](https://github.com/rr-/Tomb1Main/blob/master/CONTRIBUTING.md#coding-conventions)
- [X] I have added a changelog entry about what my pull request accomplishes, or it is an internal change

#### Description

The initial rebranding of our project from TR1Main to Tomb1Main back in 2021 was a savvy move, considering the divergent ideas and development styles between the Tomb1Main and TR2Main teams. However, the chosen name still brings to mind TR2Main, a separate project unrelated to LostArtefacts. It has become increasingly important for us to carve out our own unique brand and avoid any potential confusion in the open source community.

Hence, we have made the deliberate decision to adopt a new name: TR1X. This name embodies power, memorability, and brevity. In the vast TR fan projects landscape, we firmly believe that this name will make a lasting impression. We recognize that change can provoke mixed emotions, but we are confident that it will ultimately prove more beneficial than detrimental.

To kickstart the rebranding process, this pull request serves to update all instances of project Tomb1Main with our new project name, TR1X. Once it's approved internally, we'll continue the rebranding efforts by renaming the channels here on Discord and the GitHub repository. The old links should still work fine.

### CODE REVIEW

**Comment by rr-** [comment]
(We still need to update the logo SVG, which I plan on doing when I'm back near my main workstation.)

**Comment by walkawayy** [comment]
New .exe worked well for me on Windows. I think the `CONTRIBUTING.MD` directions could be a bit clearer for Windows at least. I had to run `docker build --progress plain . -f docker/game-win/Dockerfile -t rrdash/tr1x`.

Getting this error at the end of the `make config` output:
```
MSBUILD : error MSB1003: Specify a project or solution file. The current working directory does not contain a project or solution file.
make: *** [Makefile:71: config] Error 1
```

Similar error at the end of the `make installer` output:
```
+ dotnet restore
MSBUILD : error MSB1003: Specify a project or solution file. The current working directory does not contain a project or solution file.
make: *** [Makefile:63: installer] Error 1
```

**Comment by rr-** [comment]
Hey @walkawayy , sorry about this. `docker build --progress plain . -f docker/game-win/Dockerfile -t rrdash/tr1x` is only necessary for now and I apologize for failing to mention this in the PR. Once we merge this branch and run the GitHub Action to publid the Docker toolchain and send it to DockerHub, the current guidelines should be self-sufficient again given that Docker should download the toolchain from DockerHub automatically.

Re: `make config` and `make installer`, I'm looking into it.

**Comment by rr-** [comment]
Both C# projects seem to be building fine on my end. I improved `make clean` to remove old `bin`, `out` and `obj` artefacts from there. Could you please run `make clean config installer` and let me know if it has improved things?

**Comment by lahm86** [comment]
> Both C# projects seem to be building fine on my end. I improved `make clean` to remove old `bin`, `out` and `obj` artefacts from there. Could you please run `make clean config installer` and let me know if it has improved things?

I've tried this out and get the same error about missing the project/solution file. I noticed that both the solution files have been deleted - I think we'd need to CD into the next folder so it can see the `.csproj` file? Or going a step further, we could possibly get rid of the extra folder level in both cases, moving everything up a level - so the paths would be `tools/installer/Installer.csproj` and `tools/config/TR1X_ConfigTool.csproj`. The folder structure stems from VS assuming that there are several projects in a solution, so really not applicable in these cases.

File: tools/config/TR1X_ConfigTool/Resources/icon.ico
Comment (lahm86) [inline tools/config/TR1X_ConfigTool/Resources/icon.ico:1]: The icon looks great 👍 Should we also copy it to the same folder in the installer project so they match? The installer remains using the old one that they were both using previously.

**Comment by rr-** [comment]
I restored the .sln files – they were deleted mistakenly, as they were excluded from version control due to an entry in the .gitignore file for some reason. The build was working on my machine, because the .sln projects were there sitting around, only unversioned.

Regarding the idea of relocating the .csproj files higher up in the hierarchy, I'm uncertain if this aligns with best practices and is well-supported. My experience with C# is not sufficient to make an informed judgment. @lahm86 WDYT?

File: tools/config/TR1X_ConfigTool/Resources/icon.ico
Comment (rr-) [reply tools/config/TR1X_ConfigTool/Resources/icon.ico:1]: Yes, definitely. This was another oversight on my part. It's now done.

**Comment by lahm86** [comment]
> I restored the .sln files – they were deleted mistakenly, as they were excluded from version control due to an entry in the .gitignore file for some reason.
> 
> Regarding the idea of relocating the .csproj files higher up in the hierarchy, I'm uncertain if this aligns with best practices and is well-supported. My experience with C# is not sufficient to make an informed judgment. @lahm86 WDYT?

Ignore my suggestion. I have just realised that although you can build with just a `.csproj` file, VS will re-generate the `.sln` file always, so although we could hide it with `.gitignore` it's probably not best practice.

The projects both build for me now, thanks. And the icon looks great 👍

**Review by lahm86 — APPROVED** [review]

**Review by walkawayy — APPROVED** [review]
exe, config, and installer all work for me now. LGTM.

**Pull request merged**